target_postgres/sql_base.py

```diff
@@ -713,9 +717,19 @@ def _serialize_table_records(
                                                                      path,
                                                                      value_json_schema)
 
-                if field_name in remote_fields \
-                        and (not field_name in row
-                             or row[field_name] == NULL_DEFAULT):
+                if not field_name in row:
+                    raise Exception(
+                        "Unexpected field_name serialization! Field {} doesn't exist. Please create an issue at: {}." \
```

Comment:
"Unexpected field_name serialization!" What does this mean? The column name couldn't be generated?

Reply:
How about "Unexpected exception! Field {} doesn't exist. Please create..."

ie, this _shouldn't_ happen, and _is_ a bug.

Reply:
Actually, I think we can just get rid of this entirely. The following check to see whether the value we've placed into the row is set will blow up with a bug worthy exception. I think we can safely ignore having a pretty exception here since this should _only_ be a bug, not something we're prepared for.